apps/api/src/server/applications/application/representations/redact/redact.mapper.js

```diff
@@ -2,11 +2,22 @@ import { isEmpty, pick } from 'lodash-es';
 
 export const updateRepRedactRequestToRepository = (body) => {
 	const representation = pick(body, ['redactedRepresentation', 'redacted']);
-	const representationAction = pick(body, ['type', 'invalidReason', 'notes', 'actionBy']);
+	const representationAction = pick(body, [
+		'type',
+		'invalidReason',
+		'notes',
+		'actionBy',
+		'redactStatus'
+	]);
 
-	if (!representationAction.type) representationAction.type = 'REDACTION';
-	if (!representationAction.redactStatus) representationAction.redactStatus = true;
-	if (!representation.redacted) representation.redacted = true;
+	if ('redactStatus' in representationAction) {
```

Comment:
May be worth documenting in the router OpenAPI endpoint doc that type can be REDACT_STATUS or REDACT and that redactedRepresentation only present on REDACT.